src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Generated/SdkInfo_ComputerVisionClient.cs

```diff
@@ -1,57 +0,0 @@
-
-// <auto-generated>
-// Code generated by Microsoft (R) AutoRest Code Generator.
-// Changes may cause incorrect behavior and will be lost if the code is
```

Comment:
Please regenerate the code using generate.ps1 script, this file should not be deleted